### PR TITLE
SAK-43477 Added author id to evaluation persistence

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-43477

### CODE REVIEW

**Comment by adrianfish** [comment]
Apologies in advance for the amount of whitespace and formatting changes in this PR. Babel does stuff that it doesn't really need to be doing and I don't know how to shut that down at the moment. The formatting changes that it makes are okay, but it just makes the delta bigger, which is not ideal.

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5244563&bid=17189475)

**Comment by adrianfish** [comment]
This needs merging. Other than the whitespace, it's really simple change and is holding up Andrea.

https://jira.sakaiproject.org/browse/SAK-43455

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeSummaryTablePanel.java [hunk 1]
```diff
@@ -391,15 +391,15 @@ public boolean isVisible() {
 		});
 	}
 
-	private void addInstructorAttributeOrHide(WebMarkupContainer sakaiRubricPreviewButton, Long assignmentId, String studentId, boolean showingStudentView) {
+	private void addInstructorAttributeOrHide(WebMarkupContainer sakaiRubricButtonButton, Long assignmentId, String studentId, boolean showingStudentView) {
```
Comment (ern) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeSummaryTablePanel.java:394]: This rename probably should be sakaiRubricButton!

Comment (adrianfish) [reply gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeSummaryTablePanel.java:394]: Probably ... :)

**Pull request merged**